### PR TITLE
Movie registration view

### PR DESCRIPTION
**Pull request by TimDaub**
Questions:
- Since this is Cinemarket specific, we don't need conditionals to display the "Register Movie" button, or do we? I'm assuming we'll not try to merge this upstream in austingriffith/burner-wallet.

TODOs:

- [x] Figure out where to merge these changes in. They kinda don't belong in leapdao/burner-wallet.
- [ ] ~Have i18n files translated to other languages~
- [ ] ~Upload file and picture~
- [x] Mint a token
- [x] Make sure that scanner doesn't reset state of view
- [x] Give Receipt after minting
- [ ] ~Progress bar~
- [x] Handle TODOs in the code

The strike-through TODOs will be handled in other PRs.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "axios": "^0.18.0",
     "base64url": "^3.0.1",
     "bootstrap": "^4.1.3",
+    "buffer": "^5.2.1",
```
Comment (TimDaub) [inline package.json]: Pin

File: package.json [hunk 2]
```diff
@@ -25,6 +26,7 @@
     "helmet": "^3.14.0",
     "i18next": "^13.1.5",
     "i18next-browser-languagedetector": "^2.2.4",
+    "ipfs-http-client": "^30.1.2",
```
Comment (TimDaub) [inline package.json]:
```suggestion
    "ipfs-http-client": "30.1.2",
```

File: package.json (hunk 1 above)
Comment (TimDaub) [inline package.json]:
```suggestion
    "buffer": "5.2.1",
```

**Comment by johannbarbie** [comment]
> Figure out where to merge these changes in. They kinda don't belong in leapdao/burner-wallet.

we can have a separate branch, but deploy them some-where else?

**Comment by TimDaub** [comment]
> we can have a separate branch, but deploy them on a different domain.

Sounds good. I'll push a branch called `cinemarket` and change the base of this PR.

**Pull request merged**